src/modules/person/persistence/person.repository.ts

```diff
@@ -148,13 +150,28 @@ export class PersonRepository {
         scope.findBy({ id: personId }).sortBy('vorname', ScopeOrder.ASC);
 
         const [persons]: Counted<Person<true>> = await this.findBy(scope);
-        const person: Person<true> | undefined = persons[0];
-
+        let person: Person<true> | undefined = persons[0];
         if (!person) return { ok: false, error: new EntityNotFoundError('Person') };
+        person = await this.extendPersonWithKeycloakData(person);
 
         return { ok: true, value: person };
     }
 
+    public async extendPersonWithKeycloakData(person: Person<true>): Promise<Person<true>> {
+        if (!person.keycloakUserId) {
+            return person;
+        }
+
+        const keyCloakUserDataResponse: Result<User<true>, DomainError> = await this.kcUserService.findById(
+            person.keycloakUserId,
+        );
+        if (keyCloakUserDataResponse.ok) {
+            person.attributes = keyCloakUserDataResponse.value.attributes;
+            person.isLocked = keyCloakUserDataResponse.value.enabled === false;
```

Comment:
Like I've said in another comment, `attributes` might contain additional Data for external systems. Is probably not a problem here, but something you should be aware of

Reply:
Should not be a problem anymore, because the lock-attributes are a separate type now.